application/src/main/resources/thingsboard.yml

```diff
@@ -739,6 +739,8 @@ queue:
     compression.type: "${TB_KAFKA_COMPRESSION_TYPE:none}" # none or gzip
     batch.size: "${TB_KAFKA_BATCH_SIZE:16384}"
     linger.ms: "${TB_KAFKA_LINGER_MS:1}"
+    max.request.size: "${TB_KAFKA_MAX_REQUEST_SIZE:1048576}"
```

Comment:
These parameters should be added to the tb-*-transport.yml files

Reply:
Files tb-*-transport.yml updated. 
Kafka other properties example added (with default values and references).